packages/common/src/components/animations/AnimateIn.tsx

```diff
@@ -25,6 +25,7 @@ interface IPreventScroll {
 const AnimationContainer = styled.div``;
 
 export default function AnimateIn({
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I think we can avoid this as well by either removing the unused variable or prefixing it with an underscore.

Reply:
If `componentRef` is unused I think removing it should be fine. Let me know your thoughts on that.

Reply:
yes, that would be better

Reply:
Done ✅